tracer/test/test-applications/integrations/Samples.Kafka/Program.cs

```diff
@@ -21,8 +21,11 @@ static async Task<int> Main(string[] args)
 
                 await TopicHelpers.TryDeleteTopic(topic, config);
 
-                await ConsumeAgainstNonExistentTopic(topic, config);
+                // null message and topic were edge cases where we were throwing in our integrations
+                await TryProduceWithNullMessage(topic, config);
```

Comment:
I'm interested _where_ we were throwing, because AFAICT, we were checking topic for `string.IsNullOrEmpty` in al the code paths previously 🤔

Reply:
![image](https://github.com/user-attachments/assets/8be6619e-352e-46f4-83f1-17db63a93b68)

`message` is `null` (well I guess not based on the above comments), but the `message.Instance` is `null`
Which then causes the `message.Value` to throw here.